.gitpod.yml

```diff
@@ -0,0 +1,2 @@
+tasks:
+  - init: npm install && npm run build
```

Comment:
You could add a command phase, which gets execute after init on when you start a workspace.
```
    command: npm run serve
```

Reply:
@svenefftinge thanks for the hint. I will add it and see how it works.

I am still learning how gitpod.yml works. Currently I'm able to build and run my node app from the command line terminal in gitpod. See screenshot.

Still not quite clear how the gitpod.yml init and command lines are integrated in the UX. 

[This document](https://www.gitpod.io/docs/config-start-tasks/) is helpful. A few screenshots or videos with example workflows would be great.

Reply:
![Screen Shot 2020-11-25 at 6 19 50 PM](https://user-images.githubusercontent.com/2234901/100294812-7d883d80-2f4d-11eb-9658-8468f88d2668.png)

Reply:
I have a somewhat related question about github access. I opened this issue about it: https://github.com/gitpod-io/gitpod/issues/2287

Reply:
I have no idea why you see that message, although you can push via terminal. Maybe you can add some additional information to the ticket, that would help us to reproduce the issue.